notebooks/Sigma_test_Notebook.ipynb

```diff
@@ -138,7 +205,7 @@
    "outputs": [],
    "source": [
     "for rule in rules:\n",
-    "    rule.__dict__"
+    "    rule.id"
```

Comment:
don't want to do something more here?

```python
print(f'ID: {rule.id}')
print('something a bit more than the id?')
```
or somethign like that...

Reply:
good point. yes